modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -982,7 +982,7 @@ export class Wallets implements IWallets {
         .result();
       const multisigTypeVersion =
         tssSettings.coinSettings[this.baseCoin.getFamily()]?.walletCreationSettings?.multiSigTypeVersion;
-      if (this.baseCoin.isEVM() && multisigTypeVersion === 'MPCv2') {
+      if (!walletVersion && this.baseCoin.isEVM() && multisigTypeVersion === 'MPCv2') {
         walletVersion = 5;
       }
```

Comment:
The codes before force the `walletVersion` to be 5. Now we will allow this `walletVersion` to be set by the user. If I understand it correctly, the walletVersion here can only be 5 or 6. Should we check this here? Or do you plan to validate that later in the backend?

Reply:
backend has the validation, and overriding user input doesn't look correct to me. Similar code is there in the UI also to decide the wallet version